easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = '8.10.2'
+
+homepage = 'https://www.abinit.org/'
+description = """ABINIT is a package whose main program allows one to find the total energy,
+ charge density and  electronic structure of systems made of electrons and nuclei (molecules
+ and periodic solids) within Density Functional  Theory (DFT), using pseudopotentials and a
+ planewave or wavelet basis."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://www.abinit.org/sites/default/files/packages/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['4ee2e0329497bf16a9b2719fe0536cc50c5d5a07c65e18edaf15ba02251cbb73']
+
+dependencies = [
+    ('libxc', '3.0.1'),
+    ('netCDF', '4.6.1'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('HDF5', '1.10.2'),
+]
+
+# ensure mpi and intel toolchain
+configopts = '--enable-mpi '
+configopts += '--enable-gw-dpc '
```

Comment:
@jmbeuken is double precision really needed for GW calculations? note that it is marked 'EXPERIMENTAL'.

Reply:
The documentation is outdated.  Double precision is ok in GW
Actually, this is the option we use in our testfarm for testing. 
The only drawback is that GW in double precision is slower and more memory demanding...

Reply:
@jmbeuken can you add a comment that this option is for enabling double precision?